pandas/types/common.py

```diff
@@ -491,3 +491,27 @@ def pandas_dtype(dtype):
         return dtype
 
     return np.dtype(dtype)
+
+
+def _is_fillable_value(value):
+    pandas_ts_types = ('Timestamp', 'Period', 'Timedelta')
+    pandas_block_types = ('Series', 'DataFrame')
+
+    if any([isinstance(value, (list, dict)),
+            callable(value),
+            (not (isinstance(value, string_types) or
+                  isinstance(value, (int, float, complex, str, None.__class__)) or
+                  is_numeric_dtype(value) or
+                  is_datetime_or_timedelta_dtype(value) or
```

Comment:
the dtype must be passed in otherwise how do unknow is the filll_value is the right type
e.g. an int is not valid if u have a datetime array